store/storetest/channel_member_history_store.go

```diff
@@ -16,8 +16,9 @@ import (
 func TestChannelMemberHistoryStore(t *testing.T, ss store.Store) {
 	t.Run("Log Join Event", func(t *testing.T) { testLogJoinEvent(t, ss) })
 	t.Run("Log Leave Event", func(t *testing.T) { testLogLeaveEvent(t, ss) })
-	t.Run("Get Users In Channel At Time", func(t *testing.T) { testGetUsersInChannelAt(t, ss) })
-	t.Run("Purge History", func(t *testing.T) { testPermanentDeleteBatch(t, ss) })
+	t.Run("Get Users In Channel At Time - ChannelMemberHistory", func(t *testing.T) { testGetUsersInChannelAtChannelMemberHistory(t, ss) })
+	t.Run("Get Users In Channel At Time - ChannelMembers Fallback", func(t *testing.T) { testGetUsersInChannelAtChannelMembers(t, ss) })
+	t.Run("Permanent Delete Batch", func(t *testing.T) { testPermanentDeleteBatch(t, ss) })
```

Comment:
Probably only matters for consistency, but we use "CamelCaps" for test names everywhere else. At the very least it makes the output match the real names since the spaces aren't replaced with underscores and other characters aren't escaped.

Reply:
Fixed in https://github.com/mattermost/mattermost-server/pull/7951/commits/99e7c1aa3493582d5793dcef0336da5f021c985d